index.html

```diff
@@ -0,0 +1,104 @@
+</head>
+<body>
+
+    <nav>
+      <ul class="navbar">
+          <li><a href="index.html">Home</a></li>
+          <!--<li><a href="page2.html">page2</a></li> -->
+          <li><a href="images">page3</a></li>
+      </ul>
+    </nav>
+
+<!--Article element containing your bio-->
+	<header>
+    <h2>About Craig</h2>
+  </header>
+	  <section>
+	  	 <article>
+
+        <title>About Craig</title>
+        <h2>About Craig</h2>
+		    <p>This is the paragraph about Craig, and yes I will be talking about him in the third party becaause it is fun.<br> Not much to talk about He works as a mechanic all of his life.  At least the part that he does remember. He have been a mechanic/ computer geek all his life and he has been told.<br>
+		    He spent a month after Hurricane Katrina keeping generators going to keep the pumps running all around Mississippi and Louisiana. During the flood of Nashville he was on generator duty again the keep clean water flowing for the residents. He has seen many of hard times, but he thinks javascript is just has hard has Mother's Natures fury...<br> 
+		    Seven years ago he was hit very hard during a storm. The winds where he was working were 70 plus miles an hour. He suffered a Traumatic Brain Injury (TBI). Not much to talk about he just remembers seeing stars and waking up in the hospital a few days later. Yes in computer terms he had a format C and had his OS reloaded but many files were lost in the transfer, better word for it he had an upgrade. Now that that part of his bio is over with. Life goes on sometimes with 2 lumps.
+		    </p>
+   	
+		 <title>Life now</title> 
+		 <h2>Life now</h2>  	
+			<q>Now life goes on and he is going to do it better this time. He is going to get a degree/ diploma in computer coding. He has always wanted to do this since the 6th grade when he was first allowed to really play with computers and basic coding. He finds that it is very rewarding creating webpages and has have fun with them they may not be a good looking as big website but he is still learning. With every mistake he makes he learns twice as much. With every mistake you learn what not to do and how to fix it and can help someone else in the the future.
+			</q>
+
+		<title>My Hobbies</title>
+		<h2>My Hobbies</h2>  		
+			<q>My hobbies are as follows I like computers, building them and playing games on them. Renaissance festivals is my number two hobby, it were I can play with swords. Yes I play with swords and teach other people stage combat. Plus I also make swords so not only do I get to play with swords I made and teach stage combat. Sorry that total geek sentence run on error… Hey did you see the new Star Wars 
+			</q>
+
+		</article>
+
+		<div>
+	  		<h3>Here lies a link to a fun place to visit on the web and in real life...</h3>
+			<a href="url">http://www.tnrenfest.com/index.htm</a>
+	  		<!-- Article element with links to resources or sites you like-->	
+  		</div>
+
+  		<article>
```

Comment:
Take this comment from the specs literally "Article element with two sections for future projects (placeholders)" - (wink, use sections, wink).